Tasks/AzurePowerShellV4/Utility.ps1

```diff
@@ -1,19 +1,61 @@
+function Get-SavedModuleContainerPath {
```

Comment:
make these functions os agnostic. pass in an argument called 'platform' and then do the if-else logic inside it. use join-path cmdlet to handle windows and linux path separators ('/' or '\')